google-cloud-spanner/pom.xml

```diff
@@ -18,38 +18,9 @@
   </parent>
   <properties>
     <site.installationModule>google-cloud-spanner</site.installationModule>
-    <artifact.version>${project.version}</artifact.version>
   </properties>
   <build>
-    <resources>
-      <resource>
-        <directory>./..</directory>
-        <filtering>true</filtering>
-        <includes>
-          <include>project.properties</include>
-        </includes>
-      </resource>
-    </resources>
     <plugins>
-       <plugin>
-        <artifactId>maven-antrun-plugin</artifactId>
-         <version>1.4</version>
-         <executions>
-          <execution>
-            <phase>process-resources</phase>
-            <configuration>
-               <tasks>
-                 <copy file="${project.build.outputDirectory}/project.properties"
-                   toFile="${project.build.outputDirectory}/com/google/cloud/spanner/project.properties"
-                   overwrite="true" />
-              </tasks>
-            </configuration>
-            <goals>
-              <goal>run</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
this seems unrelated?

Reply:
This is related. This change also simplifies the way `x-goog-api-client` header is handled differently now (just gets data from `METADATA.MF` and the project.properties is not needed anymore, thus removing its generation (same cleanup was done for all other poms in this repo).